core/cat/experimental/form/cat_form.py

```diff
@@ -34,11 +34,15 @@ def __init__(self, cat) -> None:
 
         self._errors: List[str] = []
         self._missing_fields: List[str] = []
+        self.model_class = self.model_getter()
```

Comment:
isn't this a loop?

Reply:
yes, that's redudant, I'll remove it. Thank you for the review @matteocacciola!